### DISCUSSION TITLE
Going crazy

### DISCUSSION THREAD

**Discussion by daltonnoah** [TeamVanced/VancedManager#676] [Q&A]
Why can I not sign into my Vanced

**Comment by MrAppAndCrap** [comment]
> Why can I not sign into my Vanced

Have you installed microg?

**Comment by Achxy** [comment]
In order to Sign into account you need to have MicroG installed, which is available on your Vanced Manager. 


![IMG_20211020_210108.jpg](https://user-images.githubusercontent.com/74144790/138124023-de05df43-a188-419d-89a3-05d1958ec82b.jpg)
